jetty-servlet/src/test/java/org/eclipse/jetty/servlet/ErrorPageTest.java

```diff
@@ -198,6 +229,37 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response) t
         }
     }
 
+    public static class AsyncSendErrorServlet extends HttpServlet implements Servlet
+    {
+        @Override
+        protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException
+        {
+            try
+            {
+                final CountDownLatch hold = new CountDownLatch(1);
```

Comment:
Remove `final`.

Reply:
Ha! tell that to checkstyle!   Without final we get complaints that a variables declaration is too soon before it's usage.
So while final is not necessary for the compiler to work it out, apparently checkstyle wants the final to help the programmer work it out!